.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,124 @@
+name: "Focus/Firefox Tests"
+    
+on:
+    workflow_dispatch:
```

Comment:
We will change this to run on a regular basis. Now, we can dispatch this workflow on demand.

Reply:
you can leave it so that it can be triggered when needed